Backpack-SwiftUI/CarouselCard/Classes/BPKCarouselCard.swift

```diff
@@ -36,36 +36,44 @@ public struct BPKCarouselCard<Content: View>: View {
         self.title = title
         self.description = description
         self.contentAccessibilityLabel = contentAccessibilityLabel
+        UIScrollView.appearance().bounces = false
```

Comment:
why do we need this?

Reply:
will it not affect every scrollview in the app? (as it's setting a static property)

Reply:
We did this to be able to allow for the overflow of content to be shown but when content fits it doesn't act scrollable